node/packages/sdk/lib/structured-log-to-event.js

```diff
@@ -132,6 +132,11 @@ module.exports.attemptParseStructuredLogAndCapture = (logLine) => {
       case 'WARN':
         handleWarningLog(logLineParsed);
         return;
+      case 'INFO':
```

Comment:
I think there's no way we can have `INFO` returned here (?)

Reply:
You're right. I thought we were returning `info` or null.